npm-shrinkwrap.json

```diff
@@ -2,6 +2,18 @@
   "name": "etcher",
   "version": "1.0.0-beta.19",
   "dependencies": {
+    "ajv": {
```

Comment:
Wow, hundreds more dependencies. Are they all actually needed, or is `npm` playing up again?